Kitodo/src/main/java/de/sub/goobi/helper/servletfilter/RequestControlFilter.java

```diff
@@ -71,6 +73,11 @@
     @SuppressWarnings({"unchecked", "rawtypes" })
     public void init(FilterConfig config) throws ServletException {
 
+        encoding = config.getInitParameter("requestEncoding");
```

Comment:
Where must this parameter be defined to set the value? It would be nice if you added the proper value to this file.